.gitignore

```diff
@@ -13,3 +13,6 @@ target/
 Cargo.nix
 crate-hashes.json
 result
+
+.vscode/
```

Comment:
You can apply this as a user-global gitignore fwiw.

Reply:
ok, I's gone.